ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -22,12 +23,16 @@ StatusDialog {
 
     objectName: "swapModal"
 
-    implicitWidth: 556
```

Comment:
please keep this as it is

Reply:
hmm I was getting some height binding loop errors before so I played around with this, let me revert and re-check